src/Editor.jsx

```diff
@@ -11,28 +10,50 @@ class Editor extends Component {
         this.state = {
             step: 0,
             user: null,
-            alertText: null
+            alertText: null,
+            inGraph: null,
+            lastGraphVisited: null,
         };
 
         Object.assign(this.state, defaultGraph);
 
         this.backbutton = React.createRef();
```

Comment:
Our custom back button has been removed, right? I think this variable can be removed.

Reply:
Sure thing